examples/experimental/dagster-components/dagster_components/core/component_defs_builder.py

```diff
@@ -19,13 +22,48 @@
 if TYPE_CHECKING:
     from dagster._core.definitions.definitions_class import Definitions
 
+import importlib.util
+import inspect
+
+
+def load_module_from_path(module_name, path) -> ModuleType:
```

Comment:
Generated this from chatGPT. Similar to `import_module_from_path` in dagster._seven but does not interact with `sys.modules`.